src/server/services/Actions.ts

```diff
@@ -101,7 +101,7 @@ class Actions {
   }
 
   async teleportTokens(
-    dripAmount: number,
+    dripAmount: number | bigint,
```

Comment:
The input here should be `bigint` only for more robustness. Should you do this `convertAmountToBn` in

https://github.com/paritytech/substrate-matrix-faucet/blob/82728e9e91e1978dbd623c616fe8e8c53dd8c7a7/src/server/services/Actions.ts#L194

instead?